hloc/reconstruction.py

```diff
@@ -73,11 +110,9 @@ def run_reconstruction(
     if options is None:
         options = {}
     options = {"num_threads": min(multiprocessing.cpu_count(), 16), **options}
-    with OutputCapture(verbose):
-        with pycolmap.ostream():
-            reconstructions = pycolmap.incremental_mapping(
-                database_path, image_dir, models_path, options=options
-            )
+    reconstructions = incremental_mapping(
```

Comment:
Maybe define the OutputCapture here to reduce indentation in incremental_mapping?

Reply:
Nice, changed it.